### ISSUE TITLE
Content-* (Trac #103)

### ISSUE THREAD

**Issue by mnot** [httpwg/httpbis-issues#103]
What does Content-* mean? Does it mean all headers prefixed with "Content-", or just those headers prefixed by "Content-" that are defined in RFC 2616?

Migrated from https://trac.ietf.org/ticket/103
```json
{
    "status": "closed",
    "changetime": "2011-03-10T08:40:47",
    "_ts": "1299746447000000",
    "description": "What does Content-* mean? Does it mean all headers prefixed with \"Content-\", or just those headers prefixed by \"Content-\" that are defined in RFC 2616?\n\n",
    "reporter": "mnot@pobox.com",
    "cc": "",
    "resolution": "fixed",
    "time": "2008-02-28T00:27:57",
    "component": "p2-semantics",
    "summary": "Content-*",
    "priority": "",
    "keywords": "",
    "milestone": "unassigned",
    "owner": "julian.reschke@gmx.de",
    "type": "design",
    "severity": ""
}
```

**Comment by mnot** [comment]
Trac update at 20080729T23:41:35: fielding@gbiv.com commented:

> It means all header fields that strat with the prefix "Content-",
> regardless of where they were defined.
>

**Comment by mnot** [comment]
Trac update at 20081104T14:05:42: 
* julian.reschke@gmx.de changed owner from "" to "julian.reschke@gmx.de"
* julian.reschke@gmx.de changed status from "new" to "assigned"

**Comment by mnot** [comment]
Trac update at 20081104T14:18:23: 
* julian.reschke@gmx.de changed attachment from "" to "132.diff"
* julian.reschke@gmx.de commented:

> proposed change for part 2

**Comment by mnot** [comment]
Trac update at 20081104T15:00:53: 
* julian.reschke@gmx.de changed attachment from "" to "i103.diff"
* julian.reschke@gmx.de commented:

> proposed change for part 2.

**Comment by mnot** [comment]
Trac update at 20081105T10:13:34: 
* julian.reschke@gmx.de commented:

> Fixed in [333]:
> 
> Resolve #103: explain shorthand notation "Content-*" (closes #103).


* julian.reschke@gmx.de changed resolution from "" to "fixed"
* julian.reschke@gmx.de changed status from "assigned" to "closed"

**Comment by mnot** [comment]
Trac update at 20110310T08:40:47: fielding@gbiv.com commented:

> From [1158]:
> 
> Replaced the general prohibition on unrecognized Content-* header fields
> with a specific prohibition of Content-Range (the only field for which
> it is an actual problem) and a general requirement regarding checking
> for consistency.  Unfortunately, this required rewriting the entire
> section on PUT to get rid of the misconceptions about storing resources
> and reflect how PUT is actually implemented in practice.
> 
> Addresses #79, #102, #103, #104, #112, #180, #231, and #267
>
